### PR TITLE
default, server_default and creating new invitation

### PR DESCRIPTION
**Pull request by KamiTzayig**
in order for the default values to be inserted from the db when creating a new sql row you need to add default &server_default to the models.py so sqlalchemy can expect the values to be inserted, also i created the restapi for creating a new order

### CODE REVIEW

**Review by yarons — COMMENTED** [review]
There has been some changes introduced by @inbarbarkai, I want to make sure we're not conflicting, can you please take a look at hist branch and consult him about this?

**Comment by inbarbarkai** [comment]
Lets merge mine and see, it shouldn't contain too much conflicts if any.

File: src/routers/inviter_router.py [hunk 1]
```diff
@@ -9,4 +10,16 @@ def create_invitations_list():
 
 @router.get("/invitations", summary="Gets all the invitations for this user.")
 def get_invitations(invitations: InvitationRepository = Depends(create_invitations_list), user_id: str = None):
-    return invitations.query(lambda x: x.filter(Invitation.user_id == user_id))
\ No newline at end of file
+    return invitations.query(lambda x: x.filter(Invitation.user_id == user_id))
+
+@router.get("/invite", summary="Creates a new invitation for this user.")
+def create_invitation(invitations: InvitationRepository = Depends(create_invitations_list), user_id: str = None, invitees_amount: int = None, invitees_arrival_timestamp_epoch: int = None,comment_for_guard: str =  None):
+    new_invitation =  Invitation(
```
Comment (inbarbarkai) [inline src/routers/inviter_router.py:17]: Do some data validations

Comment (inbarbarkai) [reply src/routers/inviter_router.py:17]: Why are you using get to create new invite?
Use post or put.
Also, prefably, don't use query parameters for input.
Use the body

**Review by inbarbarkai — CHANGES_REQUESTED** [review]

**Comment by yarons** [comment]
@inbarbarkai I'm guessing it's no longer relevant, am I correct?
Sorry @KamiTzayig :(

**Comment by inbarbarkai** [comment]
@yarons I don't know... That is for @KamiTzayig to decide

**Comment by yarons** [comment]
@KamiTzayig we should rebase your changes, some of them are required for your UI work.

**Comment by yarons** [comment]
Conflicting, irrelevant, reopen if necessary.

**Pull request closed without merging**